Instacount Functionality

schreiaj opened the pull request.

Commit:
Working on propagating changes and rebuilding query object when changes occur

Commit:
Instacount working for patients.

Commit:
Fix query saving

noranda commented:
This needs `numeral.js` and it needs to run a query initially to get the count of all patients/conditions/encounters.

noranda commented:
Patient age filters aren't working.

Commit:
Added numerals for number formatting

Added numeral helper for formatting numbers

Updated Ember Item Changed prop to handle ranges

Forced a call to the instacount endpoints on init to fill them initially.

Commit:
Removed console.logs
